spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/TestBase.java

```diff
@@ -77,6 +77,7 @@ public static void startMetastoreAndSpark() {
             .config(SQLConf.PARTITION_OVERWRITE_MODE().key(), "dynamic")
             .config("spark.hadoop." + METASTOREURIS.varname, hiveConf.get(METASTOREURIS.varname))
             .config("spark.sql.legacy.respectNullabilityInTextDatasetConversion", "true")
+            .config("spark.files.maxPartitionBytes", "100")
```

Comment:
Is this needed? If it is for splitting tasks, I don't think it will work for Iceberg. We should use the equivalent property in Iceberg.